package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "workshop-protractor",
+  "version": "1.0.0",
+  "description": "This is a workshop about Protractor",
+  "main": "index.js",
+  "scripts": {
+    "clean": "rd /s /q dist",
```

Comment:
change to rm -rf because you will get some troubles trying to integrate with Travis